opencog/timeoctomap/TimeOctomap.cc

```diff
@@ -29,67 +29,71 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-//TimeSpaceAtom.cpp
-#include "TimeSpaceAtom.h"
+//TimeOctomap.cpp
+#include "TimeOctomap.h"
 //#include "octomap/OcTreeKey.h"
 //#include <assert.h>
 #include "opencog/util/oc_assert.h"
-TimeSpaceAtom::TimeSpaceAtom(unsigned int num_time_units,
-                             vector<double>map_res_meters):
-    time_circle(num_time_units), created_once(false)
+
+TimeOctomap::TimeOctomap(unsigned int num_time_units,
+                         double map_res_meters,
+                         duration_c time_resolution):
+    time_circle(num_time_units),map_res(map_res_meters),time_res(time_resolution),created_once(false)
```

Comment:
To avoid some gdb warning, time_circle should be initialized after map_res and time_res.